hazelcast/src/test/java/com/hazelcast/jet/core/TestProcessors.java

```diff
@@ -51,6 +53,7 @@
 import static com.hazelcast.jet.Traversers.traverseStream;
 import static com.hazelcast.jet.Util.entry;
 import static com.hazelcast.jet.core.BroadcastKey.broadcastKey;
+import static com.hazelcast.jet.core.ExecutionLifecycleTest.NON_SERIALIZABLE_EXCEPTION;
```

Comment:
Cycle in dependencies. Better to move the constant to the NonSerializableException, if you want to have a constant (IMHO it's good enough to use `new` everytime)

Reply:
Moved NonSerializableException to ExecutionLifecycleTest.java

Reply:
It's other way around - the exception should be in TestProcessors, same for constants. Otherwise you still have cycle. You can also extract this exception to upper level and create constant in the exception class, it will also remove the cycle